### PR TITLE
Enable uap F5 arm helix runs

### PR DESCRIPTION
**Pull request by joperezr**
cc: @danmosemsft @weshaggard @safern @MattGal @Chrisboh 

Adding UWP F5 ARM runs in helix. I have submitted a private run to make sure this works as expected and although the run hasn't finished, things look promising(so far 110791 tests pass, 33 failures) Correlation ID for my private job is 97552c7b-36f4-48e5-98ba-9f0ef4bcdab7

### CODE REVIEW

File: buildpipeline/pipeline.json [hunk 1]
```diff
@@ -156,11 +156,13 @@
           }
         },
         {
-          "Name": "DotNet-CoreFx-Trusted-Windows-NoTest",
+          "Name": "DotNet-CoreFx-Trusted-Windows",
           "Parameters": {
             "PB_Platform": "arm",
             "PB_BuildArguments": "-framework=uap -buildArch=arm -Release -- /p:SignType=real /p:RuntimeOS=win10",
-            "PB_SyncArguments": "-p -- /p:ArchGroup=arm /p:RuntimeOS=win10"
+            "PB_BuildTestsArguments": "-framework=uap -buildArch=arm -Release -SkipTests -- /p:RuntimeOS=win10 /p:ArchiveTests=true",
```
Comment (danmosemsft) [inline buildpipeline/pipeline.json]: -outerloop ?

**Comment by MattGal** [comment]
The UAP tests are a bit slower than the UAPAOT post-compiled tests for execution speed.

Regarding inner/outerloop, you guys can do what you want, but I'd point out right now I only have 7 physical Windows ARM machines and this will slow down overall execution.  Given official runs start @ 5 AM it may not matter.

**Comment by danmosemsft** [comment]
Any idea how long execution time typically is for outer vs inner loop on Windows ?

**Comment by MattGal** [comment]
@danmosemsft  varies by tests.  Seat-of-the pants estimate the biggest delta is likely to be in the few minutes range, but when multiplied by the # of runs that will be coming in (@wtgodbe is doing the same thing right now for CoreCLR) ARM is going to start getting backed up.

**Pull request merged**